### DISCUSSION TITLE
Python_maintainers: Please add your userid to .github/CODEOWNERS

### DISCUSSION THREAD

**Discussion by cclauss** [TheAlgorithms/Python#8571] [Team Posts]
https://github.com/TheAlgorithms/Python/blob/master/.github/CODEOWNERS has been established by @dhruvmanila to ensure that we take a divide-and-conquer approach to pull requests.  Now that Hacktoberfest has moved us from 30 open PRs to more than 300 open PRs in just a few weeks, this is more important than ever.

Please edit that file and add your GitHub userid next to all directories that you have an interest in.  This will allow us to spread the load of reviews and get our number of pull requests back down to a reasonable level.  Thanks.

**Comment by l3str4nge** [comment]
I can't write it directly. Should I create PR for that @cclauss ? I am interested in ` /digital_image_processing/` and `/sorts/`

**Comment by cclauss** [comment]
Sure.  We will approve... ;-)
